composer.json

```diff
@@ -6,7 +6,7 @@
     "license": "MIT",
     "require": {
         "php": "^8.1",
-        "cakephp/console": "^5.0",
+        "cakephp/cakephp": "^5.0",
```

Comment:
This was necessary since to make the testsuite work again since a router reference was added in the core package in [this PR](https://github.com/cakephp/cakephp/pull/17639/files#diff-35d4aaeff519e937f2c0abbd566b15c1bd21c8e26ba5eea56fe7fbfb0d68b15dR83)

Reply:
There are even more problems related to the `cakephp/console` package.
https://github.com/cakephp/cakephp/issues/18104